docs/usage/types/custom.md

```diff
@@ -255,11 +261,19 @@ model = Model(
     thing=TastingModel(name='Python', aged='Not much', quality='Awesome'),
 )
 print(model)
+"""
+wine=<__main__.TastingModel object at 0x0123456789ab> cheese=<__main__.TastingModel object at 0x0123456789ab> thing=<__main__.TastingModel object at 0x0123456789ab>
```

Comment:
whoa how does it know to use that memory address? I was going to suggest changing the repr but looking at it and seeing there is a pattern makes me think something smart is happening here and it's useful for reinforcing that it's a custom class. 🤯

Reply:
I don't know how it would work haha, I just changed it to a dataclass so the repr is nice now